server/README.adoc

```diff
@@ -28,6 +28,10 @@ KROKI_DOT_BIN_PATH:: Path to the `dot` binary (provided by GraphViz)
 KROKI_NOMNOML_BIN_PATH:: Path to the `nomnoml` binary
 KROKI_BLOCKDIAG_HOST:: The host of the blockdiag server
 KROKI_BLOCKDIAG_PORT:: The port of the blockdiag server
+KROKI_SAFE_MODE:: Either `SECURE` (default) or `UNSAFE`. Determines whether to sanitize requests for formats that allow including external content
+KROKI_PLANTUML_INCLUDE_PATH:: The include path to set for PlantUML.
+KROKI_PLANTUML_INCLUDE_WHITELIST:: The name of a file that consists of a list or regexes for valid includes.
```

Comment:
I guess that passing a list of regexes as environment variable is not practical but using a file means that you need to extend the docker image or bind mount a directory with this file right?
Is there an alternative solution?

Reply:
you could have one regex per env-var and chase up env vars with indexes, e.g. `KROKI_PLANTUML_INCLUDE_WHITELIST_0`, `KROKI_PLANTUML_INCLUDE_WHITELIST_1`, until you stop finding `KROKI_PLANTUML_INCLUDE_WHITELIST_n` for some integer n